### PR TITLE
bug/UI inconsistencies

### PR DESCRIPTION
**Pull request by Gnzlt**
## What does this implement/fix? Explain your changes.
- [AND-12764](https://jira.developers.mega.co.nz/browse/AND-12764)
- [AND-12773](https://jira.developers.mega.co.nz/browse/AND-12773)
- [AND-12769](https://jira.developers.mega.co.nz/browse/AND-12769)
- [AND-12767](https://jira.developers.mega.co.nz/browse/AND-12767)

## Any relevant logs, error output, etc?
...

## Any other comments?

> On the CD view
> Light & Dark theme:
> — Appbar without subtitle
> Title: CLOUD DRIVE ( so user is reassured that he/she is on the chosen destination)
> Subtitle: Select destination
> — The area comprising the buttons is grey instead of white. 
> — Buttons: text button (CANCEL) and raised button ( SAVE) since we came from the save scan view

This is being fixed in the following [MR](https://code.developers.mega.co.nz/mobile/android/android/-/merge_requests/1770).

## Where has this been tested?

- **Devices/Simulators:** 
Google Pixel 4

- **OS Version:** 
Android 11

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,8 +6,9 @@ buildscript {
         minSdkVersion = 21
         group = "com.github.meganz"
 
-        kotlinVersion = "1.4.21"
-        navigationVersion = "2.3.3"
+        kotlinVersion = "1.4.31"
```
Comment (Piasy) [inline build.gradle:9]: Should we use the same version as MEGA app repo?

**Comment by Piasy** [comment]
Everything looks fine, just small feedback.

And what's the expected behavior of https://jira.developers.mega.co.nz/browse/AND-12773 ? Currently it's still not cancellable while saving.

**Comment by Gnzlt** [comment]
> And what's the expected behavior of https://jira.developers.mega.co.nz/browse/AND-12773 ? Currently it's still not cancellable while saving.

It has been replaced by a [Progress Dialog](https://meganz.invisionapp.com/d/#/console/13743399/446859883/) which is not cancelable.

**Review by Piasy — APPROVED** [review]

**Comment by javiergm1983** [comment]
@Gnzlt I have changed the base branch because it should be `develop` like for any other PR.
@Piasy please review and approve again.

**Review by Piasy — APPROVED** [review]

**Review by javiergm1983 — APPROVED** [review]

**Review by ash-mega — CHANGES_REQUESTED** [review]
Hi @Gnzlt , the code changes are fine and other bugs are fixed except [this one](https://jira.developers.mega.co.nz/browse/AND-12767).
Follow the steps provided by Eve, the issue remains,could you please confirm? Cheers.

**Comment by Gnzlt** [comment]
> Hi @Gnzlt , the code changes are fine and other bugs are fixed except [this one](https://jira.developers.mega.co.nz/browse/AND-12767).
> Follow the steps provided by Eve, the issue remains,could you please confirm? Cheers.

Hey @ash-mega, I've just checked and it works as expected. Please make sure you are using the latest version from this branch when building.
```
implementation 'com.github.meganz:AndroidDocumentScanner:96c16ff4b4'
```

https://user-images.githubusercontent.com/5398754/114859764-7a51f900-9deb-11eb-8273-3959a0af3825.mp4

**Comment by ash-mega** [comment]
Hi @Gnzlt , you're right, the issue is fixed. All good, approved.

**Review by ash-mega — APPROVED** [review]

**Pull request merged**